File: t-tasks/src/main/java/com/teo/ttasks/api/entities/PersonResponse.java
package com.teo.ttasks.api.entities;

import com.google.gson.annotations.Expose;

public class PersonResponse {

    @Expose
    public Image image;

    @Expose
    public Cover cover;

    public static class Email {

        @Expose
        public String value;
    }

    public static class Image {

        @Expose
        public String url;
    }

    public static class Cover {

        @Expose
        public CoverPhoto coverPhoto;

        public static class CoverPhoto {

            @Expose
            public String url;
        }
    }
}

File: t-tasks/src/main/java/com/teo/ttasks/api/entities/PersonResponse.kt
package com.teo.ttasks.api.entities

import com.google.gson.annotations.Expose

class PersonResponse {

    @Expose
    lateinit var image: Image

    @Expose
    var cover: Cover? = null

    class Email {

        @Expose
        var value: String? = null
    }

    class Image {

        @Expose
        lateinit var url: String
    }

    class Cover {

        @Expose
        var coverPhoto: CoverPhoto? = null

        class CoverPhoto {

            @Expose
            var url: String? = null
        }
    }
}
